db/migrate/20210701201156_change_unrecognized_appellant_current_version.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeUnrecognizedAppellantCurrentVersion < ActiveRecord::Migration[5.2]
+  def up
+    UnrecognizedAppellant.where(current_version: nil).where(created_by: nil)
+    .all.each do |unrecognized_appellant|
+      unrecognized_appellant.update(
+        current_version: unrecognized_appellant,
+        created_by: unrecognized_appellant.claimant.decision_review.intake.user
+      )
```

Comment:
And to optimize into 1 query:
```suggestion
    .update_all(
        current_version: unrecognized_appellant,
        created_by: unrecognized_appellant.claimant.decision_review.intake.user
      )
```
There are some things to be aware of: https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/update_all

Reply:
@yoomlam How would this new code work? `unrecognized_appellant` passed into the block as a reference to each record, and I don't see how that variable would be available when using `.update_all`.

Reply:
Good point! Nevermind.